util/CMakeLists.txt

```diff
@@ -153,6 +155,10 @@ if(BUILD_UTIL_MON)
   add_subdirectory(Radiance_Monitor)
 endif()
 
+if(BUILD_UTIL_RAD)
+  add_subdirectory(Radiance_Monitor)
+endif()
```

Comment:
`Radiance_Monitor` is being built with the conditional `BUILD_UTIL_MON`, and will conflict with it on L155.

@EdwardSafford-NOAA 
Is there a special need for building just the `Radiance_Monitor`?

If so, I suggest, deprecating `BUILD_UTIL_MON` and replace them with 3 separate options; one each for Conventional, Ozone and Radiance as `BUILD_UTIL_CONMON`, `BUILD_UTIL_OZNMON` and `BUILD_UTIL_RADMON` respectively.